controller/internal/enforcer/acls/acl_test.go

```diff
@@ -180,8 +180,8 @@ func TestObservedLookup(t *testing.T) {
 			err := a.addRule(r)
 			So(err, ShouldBeNil)
 		}
-		a.reverseSort()
-		So(len(a.prefixLenMap), ShouldEqual, rulesWithObservationPrefixLens)
+		//a.reverseSort()
+		///So(len(a.prefixLenMap), ShouldEqual, rulesWithObservationPrefixLens)
 
```

Comment:
let's read the cache and keep the validations in the tests. Just in case.

Reply:
Added the cache lookups after insertion here.